applications/nrf5340_audio/README.rst

```diff
@@ -13,6 +13,19 @@ It is developed for use with the :ref:`nrf53_audio_app_dk`.
 In its default configuration, the application requires the :ref:`LC3 software codec <nrfxlib:lc3>`.
 The application also comes with various tools, including the :file:`buildprog.py` Python script that simplifies building and programming the firmware.
 
+.. _nrf53_audio_app_overview_features:
+
+Feature support matrix
+   The following table lists features of the nRF5340 Audio application and their respective limitations and maturity level.
```

Comment:
To me it looks strange that this text is "tabbed out"

Reply:
It is required to have the maturity table included at the very beginning of the doc and avoid breaking the structure of the page.